raiden/tests/scenarios/bf1_basic_functionality.yaml

```diff
@@ -66,129 +69,141 @@ scenario:
             - transfer: {from: 3, to: 0, amount: 1}
-      - parallel:
-          name: "Assert after 10 payments from D to A"
+      - serial:
+          name: "Assert after 10 payments from 3 to 0"
           tasks:
-            - assert_sum: {node: 0, balance_sum: 5010}
-            - assert_sum: {node: 3, balance_sum: 4990}
+            - wait: 10
+            - assert_sum: {from: 0, balance_sum: 2010}
+            - assert_sum: {from: 3, balance_sum: 1990}
       - serial:
-          name: "Make 10 transfers from B to E"
+          name: "Make 10 transfers from 1 to 4"
           repeat: 10
           tasks:
             - transfer: {from: 1, to: 4, amount: 1}
-      - parallel:
-          name: "Assert after 10 payments from B to E"
+      - serial:
+          name: "Assert after 10 payments from 1 to 4"
           tasks:
-            - assert_sum: {node: 4, balance_sum: 5010}
-            - assert_sum: {node: 1, balance_sum: 4990}
+            - wait: 10
+            - assert_sum: {from: 1, balance_sum: 1990}
+            - assert_sum: {from: 4, balance_sum: 2010}
       - serial:
           name: "Check that IOUs exist after the payments"
           tasks:
+            # Add a wait until all ious are processed correctly
+            - wait: 10
             - assert_pfs_history: {source: 3, target: 0, request_count: 11}
             - assert_pfs_iou: {source: 3, amount: 1100}
             - assert_pfs_history: {source: 1, target: 4, request_count: 10}
             - assert_pfs_iou: {source: 1, amount: 1000}
             # Make sure that a mediating node has not used the PFS
             - assert_pfs_iou: {source: 2, iou_exists: false}
       - serial:
-          name: "Withdraw 10% of the deposit of node C"
+          name: "Withdraw 10% of the deposit of node 2"
           tasks:
             - withdraw: {from: 2, to: 3, total_withdraw: 100, expected_http_status: 200}
       - parallel:
           name: "Assert after withdraw"
           tasks:
-            - assert: {from: 2, to: 3, total_deposit: 900, state: "opened"}
+            - assert: {from: 2, to: 3, total_deposit: 1000, total_withdraw: 100, state: "opened"}
       - serial:
-          name: "Make payments from C to E after withdraw"
+          name: "Make payments from 2 to 4 after withdraw"
           repeat: 10
           tasks:
             - transfer: {from: 2, to: 4, amount: 1}
-      - parallel:
-          name: "Assert after 10 payments from C to E"
+      - serial:
+          name: "Assert after 10 payments from 2 to 4"
           tasks:
-            - assert_sum: {node: 2, balance_sum: 4890}
-            - assert_sum: {node: 4, balance_sum: 5020}
+            - wait: 10
+            - assert_sum: {from: 2, balance_sum: 1890}
+            - assert_sum: {from: 4, balance_sum: 2020}
       - serial:
-          name: "Make payments from A to C after withdraw"
-          repeat: 10
+          name: "Make payments from 0 to 2 after withdraw"
+          repeat: 5
```

Comment:
Either change this back to `10` or change the comment below to `5`.

Reply:
Probably better to just change the comment, since you already adjusted the numbers further down too

Reply:
ok will do. Well spoted